check-sql.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Verify the the working tree modifices no sql files relative to the main
+# branch. This will always pass on a deploy because the working tree is an
+# unmodified copy of the main branch.
+
+target_treeish=${HAIL_TARGET_SHA:-$(git merge-base main HEAD)}
+
+modified_sql_file_list=$(mktemp)
+
+git diff --name-status $target_treeish sql \
+    | grep -Ev '^A|^M\t[^/]+/sql/estimated-current.txt' \
```

Comment:
This didn't work for me:

```
wmecc-475:hail jigold$ git diff --name-status $target_treeish monitoring/sql | grep -Ev '^A|^M\t[^/]+/sql/estimated-current.txt'
M	monitoring/sql/estimated-current.sql
M	monitoring/sql/initial.sql
```

Reply:
Why is initial.sql modified?

Reply:
I renamed everything to `.sql` and updated this PR, once https://github.com/hail-is/hail/pull/9560 merges this will work.

Reply:
I was doing some tests to make sure it worked.